promsonnet/prom.libsonnet

```diff
@@ -22,9 +35,9 @@
         name: name,
         rules_map:: {},
         rules_order:: [],
+        patches:: [],
         local rules_map = self.rules_map,
         local rules_order = self.rules_order,
-        rules: [rules_map[rule] for rule in rules_order],
       },
 
       rule: {
```

Comment:
`newAlert` should not be nested, just say `newAlertRule` at the same level as `pathRule`, dito for `newRecording`

wdyt?

Reply:
done